usr/share/rear/prep/BACULA/default/400_prep_bacula.sh

```diff
@@ -1,6 +1,19 @@
 ### prepare stuff for BACULA
+if ! test -d "$BACULA_CONF_DIR" ; then
+    test -d "/etc/bacula" && BACULA_CONF_DIR="/etc/bacula"
+    # bacula-enterprise-client uses /opt/bacula/etc
+    test -d "/opt/bacula/etc" && BACULA_CONF_DIR="/opt/bacula/etc"
+fi
+test -d "$BACULA_CONF_DIR" || Error "No BACULA_CONF_DIR"
+if ! test -d "$BACULA_BIN_DIR" ; then
+    BACULA_BIN_DIR="/usr/sbin"
+    # bacula-enterprise-client uses /opt/bacula/bin
+    test -d "/opt/bacula/bin" && BACULA_BIN_DIR="/opt/bacula/bin"
+fi
+export PATH=$PATH:$BACULA_BIN_DIR
 CLONE_GROUPS+=( bacula )
 COPY_AS_IS+=( "${COPY_AS_IS_BACULA[@]}" )
+COPY_AS_IS_BACULA+=( $BACULA_CONF_DIR )
```

Comment:
I think the ordering is wrong here.
I think `COPY_AS_IS_BACULA+=( $BACULA_CONF_DIR )`
should happen before `COPY_AS_IS+=( "${COPY_AS_IS_BACULA[@]}" )`
to get BACULA_CONF_DIR contents into the recovery system via COPY_AS_IS.